### ISSUE TITLE
lost-grid compatibility

### ISSUE THREAD

**Issue by FranzSkuffka** [corysimmons/typographic#50]
`@lost gutter 60px` can not be used with `vr-block`. Makes sense in next version.

**Comment by corysimmons** [comment]
You mean accepting `vr-block` as a value for gutter? Like `@lost gutter vr-block;` ?

**Comment by FranzSkuffka** [comment]
exactly

**Comment by corysimmons** [comment]
Will consider it with my new grid and when I'm converting this to PostCSS. I see your point though :+1:

**Comment by corysimmons** [comment]
I can't remember, is `vr-block` a function that returns a value?

If so, could you do something like `$sass-var: vr-block(); @lost gutter $sass-var;`?

**Comment by FranzSkuffka** [comment]
Yes. `vr-block(2)` for example.
I only remember using lost with typographic and stylus where I could not interpolate the variable into the postcss control string if I recall this correctly.

**Comment by corysimmons** [comment]
Hmm. Yeah Stylus isn't particularly great at interpolating...

Still though, I'd imagine if you did something like:

```stylus
$my-var = vr-block(2)

@lost gutter $my-var
```

And parsed over it with https://github.com/seaneking/poststylus it would work?

**Comment by corysimmons** [comment]
Tbh, I'm not sure how to do this otherwise. at-rule settings typically just accept a value. There's no real way to control it on the PostCSS side.

**Comment by FranzSkuffka** [comment]
Just mentioning this. Should be no problem in the coming version. Did you create a repo yet?

**Comment by corysimmons** [comment]
No, haven't started porting this over yet. Should launch around mid March.

**Comment by FranzSkuffka** [comment]
bump

**Comment by corysimmons** [comment]
Yeah, I've figured out some really nice vertical rhythm stuff, and even some really nice scaling stuff between headings.

I'm in the process of debating with myself about what I feel about modular scale.... Not particularly sold on it right this second. What do you think about it?

Once I come to a conclusion on that I can start combining all these parts (shouldn't take too long).

**Comment by FranzSkuffka** [comment]
It would be much easier for me to articulate my feelings if I knew about the nice stuffs you found! MS seemed solid to me but I don't know about any alternatives. Can you update me?

**Comment by corysimmons** [comment]
- http://codepen.io/corysimmons/pen/zqKJqv
- http://codepen.io/corysimmons/pen/grLWYZ

Note how the headings size in the first pen is related to the `$min/max-heading-size` settings with steps generated in between.

I dunno... I guess I could do that step thing with modular scale. :\

Maybe it's vertical rhythm I don't like... There's no way to do it correctly. Good looking vertical rhythm seems like a myth that I need to stop screwing around with.

**Comment by FranzSkuffka** [comment]
I can not really wrap my head around it right now. I sent you a doc via E-Mail that may help though.
